<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WTagEditor</class>
 <widget class="QWidget" name="WTagEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File tag edit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lFilename">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Filename:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFilename"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetFilename">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Comment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="5">
        <widget class="QComboBox" name="cbGenre">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Track:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtTrack">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="txtBPM">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="txtYear">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Artist</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Album</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLineEdit" name="txtArtist"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Year:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLineEdit" name="txtTitle"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Genre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="5">
        <widget class="QLineEdit" name="txtAlbum"/>
       </item>
       <item row="6" column="0" colspan="6">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>127</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1" colspan="5">
        <widget class="QTextEdit" name="txtComment">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>BPM:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QTreeWidget" name="twFrames">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frmAdvancedTag">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabComments">
      <attribute name="title">
       <string>Comments</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtCommentDesc"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="txtCommentLang"/>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QTextEdit" name="txtAdvComment">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QTreeWidget" name="twComments">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="btnContainer" native="true">
     <layout class="QHBoxLayout" name="btnHLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotOKClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtTrack</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotTrackChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtYear</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotYearChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtArtist</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotArtistChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenre</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotGenreChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotTitleChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtAlbum</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotAlbumChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnResetFilename</sender>
   <signal>clicked()</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotResetFilename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtComment</sender>
   <signal>textChanged()</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotCommentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBPM</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotBPMChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtFilename</sender>
   <signal>textChanged(QString)</signal>
   <receiver>WTagEditor</receiver>
   <slot>slotFilenameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOKClicked()</slot>
  <slot>slotCancelClicked()</slot>
  <slot>slotFilenameMenuPopup(QPoint)</slot>
  <slot>slotArtistChanged(QString)</slot>
  <slot>slotTitleChanged(QString)</slot>
  <slot>slotAlbumChanged(QString)</slot>
  <slot>slotGenreChanged(QString)</slot>
  <slot>slotCommentChanged()</slot>
  <slot>slotTrackChanged(QString)</slot>
  <slot>slotYearChanged(QString)</slot>
  <slot>slotResetFilename()</slot>
  <slot>slotBPMChanged(QString)</slot>
  <slot>slotFilenameChanged(QString)</slot>
 </slots>
</ui>
